<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDig</class>
 <widget class="QDialog" name="LoginDig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>232</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>352</width>
    <height>232</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>352</width>
    <height>232</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="windowOpacity">
   <double>0.950000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>9</pointsize>
     <weight>10</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(47, 47, 47);
color: rgb(255, 255, 255);
font: 87 9pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>LOGIN</string>
   </property>
  </widget>
  <widget class="QLabel" name="Userlabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 9pt &quot;Arial Black&quot;;
color: rgb(102, 102, 102);</string>
   </property>
   <property name="text">
    <string>USER NAME</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(86, 86, 86);
font: 9pt &quot;Calibri&quot;;</string>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwdLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 8pt &quot;Agency FB&quot;;
border-color: rgb(86, 86, 86);
</string>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="pwdlabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 9pt &quot;Arial Black&quot;;
color: rgb(102, 102, 102);</string>
   </property>
   <property name="text">
    <string>PASSWORD</string>
   </property>
  </widget>
  <widget class="QLabel" name="bgLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.761364, y1:0.08, x2:1, y2:1, stop:0 rgba(0, 78, 152, 255), stop:1 rgba(136, 255, 255, 255));
border-left-style:solid;border-left-width:1px;border-left-color:black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="bgLabel_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 87 7pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Welcome to traffic monitoring system</string>
   </property>
  </widget>
  <widget class="QLabel" name="logoLabel">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>1</y>
     <width>130</width>
     <height>160</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>130</width>
     <height>160</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>130</width>
     <height>160</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>0</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 127, 190);
color: rgb(255, 255, 255);
font: 87 9pt &quot;Arial Black&quot;;
border-top-style:solid;border-top-width:1px;border-top-color:black;</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <zorder>bgLabel</zorder>
  <zorder>loginButton</zorder>
  <zorder>Userlabel</zorder>
  <zorder>userLineEdit</zorder>
  <zorder>pwdLineEdit</zorder>
  <zorder>pwdlabel</zorder>
  <zorder>bgLabel_2</zorder>
  <zorder>logoLabel</zorder>
  <zorder>pushButton</zorder>
 </widget>
 <tabstops>
  <tabstop>userLineEdit</tabstop>
  <tabstop>pwdLineEdit</tabstop>
  <tabstop>loginButton</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>LoginDig</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>12</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
